frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -281,8 +282,96 @@ export function AnsibleAutomationsForm(props: {
     }
     return curator
   }
+
+  const jobTypes = [{ type: 'install' }, { type: 'upgrade' }, { type: 'scale' }, { type: 'destroy' }]
+
```

Comment:
We have a recoil selector to get the current supported Ansible job types. Probably no need to repeat them out here.

Reply:
cool, using now